modules/metrics/mule-metrics-exporter-api/src/main/java/module-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+/**
+ * Definitions for Mule Metric exporter.
+ *
+ * @moduleGraph
+ * @since 4.5
```

Comment:
I set 4.5 to everything since I believe this will be the version we'll end up releasing this in, right?

Reply:
Yes, we have to change all those.